google-datacatalog-qlik-connector/src/google/datacatalog_connectors/qlik/scrape/authenticator.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import requests
+from requests_ntlm import HttpNtlmAuth
+
+from . import constants
+
+
+class Authenticator:
+
+    @classmethod
+    def get_qps_session_cookie_windows_auth(cls, domain, username, password,
```

Comment:
I believe it does make sense to let it clear for users that the session cookie is generated by QPS. Since the Qlik solution has some components that talk to each other, I think it helps to clarify which component we are using to get the job done. Agree?